alma8/Dockerfile

```diff
@@ -36,9 +36,9 @@ RUN dnf install -y bash && bash /tmp/rpm-pkg-info.sh BASE &&\
     echo el8 > /etc/cmsos &&\
     dnf clean all
 
-ADD krb5.conf /etc/krb5.conf
-ADD ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt
-ADD fix_ssh_config.sh /tmp/fix_ssh_config.sh
+ADD @DOCKER_NAME@/krb5.conf /etc/krb5.conf
+ADD @DOCKER_NAME@/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.crt
+ADD @DOCKER_NAME@/fix_ssh_config.sh /tmp/fix_ssh_config.sh
```

Comment:
these are common scripts/configurations and can also go in some common directory e.g. `etc` or `share`. So I would suggest to copy these to `share` and then use `share/filename` in the Docker file